peddy/latest/Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
 
-MAINTAINER Xiao Dang (dangx@email.chop.edu)
+LABEL maintainer="Alex Sickler (sicklera@email.chop.edu)"
 
-RUN apt-get update --fix-missing && apt upgrade -y &&  apt-get install -y wget python bzip2 git make sysstat g++ gcc zlib1g-dev zlib1g cmake build-essential libncurses5-dev libbz2-dev liblzma-dev libcurl4-gnutls-dev libcrypto++-dev libbamtools-dev libssl-dev python-pip python-dev build-essential \
-&& git clone https://github.com/brentp/peddy && cd /peddy && pip install numpy==1.14.5 && pip install cyvcf2==0.8.9 && pip install -r requirements.txt && pip install --editable . \
-&& apt-get clean build-essential libncurses5-dev libbz2-dev liblzma-dev libcurl4-gnutls-dev libcrypto++-dev libbamtools-dev libssl-dev && apt-get autoremove -y
+RUN apt-get update && apt upgrade -y \
+&& apt-get install -y python python3-pip git \
+&& git clone https://github.com/brentp/peddy && cd /peddy && pip3 install -r requirements.txt && cd /peddy && pip3 install -r requirements.txt && pip3 install . \
```

Comment:
Pardon my ignorance, but you `cd /peddy && pip3 install -r requirements.txt` twice?

Reply:
Good catch, I missed that. But, the latest version was reverted so it doesn't matter now.